assets/css/styles.scss

```diff
@@ -693,4 +693,8 @@ div .algolia-autocomplete {
   }
 }
 
+#algolia-autocomplete-listbox-0 {
```

Comment:
Can we use something related to e.g. `search-bar` instead? This feels like an implementation detail that may change.

Reply:
I don't think Algolia is going to change this anytime soon, but I agree we can choose a better selector.